packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.2.7
+
+* Add runWithOptions entrypoint to allow external libraries to use the pigeon easier
```

Comment:
Make sure you check out the style guide for changelogs https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

Reply:
done